### PR TITLE
Add GUI solo mining guide

### PR DESCRIPTION
**Pull request by SamsungGalaxyPlayer**
I believe I made this guide correctly. This pull request contains the following changes:

1. A new GUI solo mining user guide
2. A link to the guide on the user guides page
3. Several images in the PNG folder that are referenced
4. An update to the prove payment user guide to include information about how to get the transaction key in the GUI

### CODE REVIEW

File: resources/user-guides/solo_mine_GUI.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title-kick: ""
+title-post-kick: ""
+kick-class: ""
+icon: "icon_userguides"
+attribution: "<!-- Icon is based on work by Freepik (http://www.freepik.com) and is licensed under Creative Commons BY 3.0 -->"
+---
+
+It is very easy to solo mine with the official GUI. If you have not done so already, go to the <a href="/downloads/">Monero downloads page</a> and download the official GUI for your operating system. Then, run the setup and be patient as Monero synchronizes with the network. You should see that it displays "Connected" in the lower left corner.
+
+<img src="/user-guides/png/solo_mine_GUI/01.png" style="width: 600px;"/>
```
Comment (erciccione) [inline resources/user-guides/solo_mine_GUI.md]: The correct path should be `<img src="/png/solo_mine_GUI/01.png"`

File: resources/user-guides/solo_mine_GUI.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Click on the "Advanced" tab. You should see that several other options appear. Now click on the "Mining" sub-tab.
+
+<img src="/user-guides/png/solo_mine_GUI/02.png" style="width: 600px;"/>
```
Comment (erciccione) [inline resources/user-guides/solo_mine_GUI.md]: `/user-guides` should be removed from the path

File: resources/user-guides/solo_mine_GUI.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+You should now have an option to start mining. You can optionally change the number of threads to mine with. For optimal efficiency, you should mine with your CPU's cache divided by 2. You will need to look up your CPU's specs on the manufacturer's website. If you are unsure, leave the number of threads at 1. Click the "Start mining" button.
+
+<img src="/user-guides/png/solo_mine_GUI/03.png" style="width: 600px;"/>
```
Comment (erciccione) [inline resources/user-guides/solo_mine_GUI.md]: `/user-guides `should be removed from the path

**Comment by SamsungGalaxyPlayer** [comment]
I updated the paths for the images.

**Comment by erciccione** [comment]
Sorry I forgot to mention the `/` at the beginning, that should be removed aswell. By the way, i searched some more info and looks like the better way to avoid that the link breaks in the future (if the page is moved) is to use a relative path from the domain, Given this, all three should be repaced with the relative path: `resources/user-guides/png/solo_mine_GUI/*picture.png*`

**Comment by SamsungGalaxyPlayer** [comment]
@erciccione this should be updated now. I can see the images.

**Comment by fluffypony** [comment]
@SamsungGalaxyPlayer plz rebase

**Comment by SamsungGalaxyPlayer** [comment]
@fluffypony does this work now? I updated the index.

**Comment by fluffypony** [comment]
@SamsungGalaxyPlayer perfect, thanks :)

**Pull request merged**